src/models/MySpace.test.ts

```diff
@@ -74,6 +74,24 @@ describe('My Space model', () => {
     expect(all).toHaveLength(3)
   })
 
+  it('can add a Featured Shortcut widget', async () => {
+    const created = (await MySpaceModel.addWidget(
+      {
+        userId: testUserId,
+        title: 'Featured Shortcuts',
+        type: 'FeaturedShortcuts',
+      },
+      { db }
+    )) as Widget
+
+    expect(created.title).toEqual('Featured Shortcuts')
+    expect(created.type).toEqual('FeaturedShortcuts')
+    expect(created).toHaveProperty('_id')
+
+    const all = await MySpaceModel.get({ userId: testUserId }, { db })
+    expect(all).toHaveLength(4)
+  })
+
```

Comment:
When I was adding the error state test for `ThemeToggle` I stumbled upon some missing lines in `MySpace.ts` that were easy to write tests for. Since my other attempt at writing a test didn't bump the coverage, I added a couple of tests to this file that did.